### PR TITLE
ISSUE #3504 - Revert viewer navbar opacity

### LINKED ISSUE #3504: Viewer canvas does not stretch behind navbar
### Description
- Revert #3232
- This change caused problems. Reverting this will solve #3456 and #3457

### Goals
- [ ] Be able to access the viewer without elements being obscured by the navbar

### Tasks
- [ ] Revert opacity of navbar to 1
- [ ] Prevent canvas from extending behind navbar

### PR DESCRIPTION
**Pull request by The-Daniel**
This fixes #3504 

#### Description
- Navbar is now fully opaque
- Viewer canvas does not extand behind navbar
- Exit focus mode button is now clickable (it turns out this wasn't a result of the change to the navbar and must have existed beforehand #3457)

#### Test cases
- Open the viewer
- Open windows
- Turn on the memory settings
- Open focus mode

### CODE REVIEW

File: frontend/src/v5/ui/components/shared/appBar/appBar.styles.ts [hunk 1]
```diff
@@ -16,13 +16,6 @@
  */
 
 import styled from 'styled-components';
-import { AppBar } from '@mui/material';
-
-export const AppBarContainer = styled(AppBar)`
-	@media print {
-		display: none;
-	}
-`;
```
[the remark this answers is not in the document]
Comment (sanmont3drepo) [reply frontend/src/v5/ui/components/shared/appBar/appBar.styles.ts:25]: This is how its looking in this pr
image

File: frontend/src/v5/ui/components/legal/LegalLayout/legalAppBar/legalAppBar.component.tsx [hunk 2]
```diff
@@ -29,7 +30,7 @@ type ILegalAppBar = {
 };
 
 export const LegalAppBar = ({ activePage }: ILegalAppBar): JSX.Element => (
-	<AppBarContainer position="static">
+	<MuiAppBar position="static">
```
Comment (sanmont3drepo) [inline frontend/src/v5/ui/components/legal/LegalLayout/legalAppBar/legalAppBar.component.tsx:33]: I think this creates the printing bug

File: frontend/src/v5/ui/components/shared/appBar/appBar.component.tsx [hunk 3]
```diff
@@ -14,21 +14,22 @@
...
-import { Items, AppBarContainer } from './appBar.styles';
+import { Items } from './appBar.styles';
 import { BreadcrumbsRouting } from '../breadcrumbsRouting/breadcrumbsRouting.component';
 
 export const AppBar = (): JSX.Element => {
 	const user = CurrentUserHooksSelectors.selectCurrentUser();
 
 	return (
-		<AppBarContainer position="static">
+		<MuiAppBar position="static">
```
Comment (sanmont3drepo) [inline frontend/src/v5/ui/components/shared/appBar/appBar.component.tsx:32]: check out printing

File: frontend/src/v5/ui/components/dashboard/dashboardViewerLayout/dashboardViewerLayout.component.tsx [hunk 4]
```diff
@@ -17,7 +17,7 @@
 import { ThemeProvider } from 'styled-components';
```
Comment (sanmont3drepo) [inline frontend/src/v5/ui/components/dashboard/dashboardViewerLayout/dashboardViewerLayout.component.tsx:17]: I think you can delete this one

**Review by sanmont3drepo — CHANGES_REQUESTED** [review]

File: frontend/src/v5/ui/components/shared/appBar/appBar.styles.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (The-Daniel) [reply frontend/src/v5/ui/components/shared/appBar/appBar.styles.ts:25]: Ah yeah...

File: frontend/src/v5/ui/components/dashboard/dashboardViewerLayout/dashboardViewerLayout.component.tsx (hunk 4 above)
Comment (The-Daniel) [reply frontend/src/v5/ui/components/dashboard/dashboardViewerLayout/dashboardViewerLayout.component.tsx]: deleted

File: frontend/src/v5/ui/components/legal/LegalLayout/legalAppBar/legalAppBar.component.tsx (hunk 2 above)
Comment (The-Daniel) [reply frontend/src/v5/ui/components/legal/LegalLayout/legalAppBar/legalAppBar.component.tsx]: fixed

**Review by sanmont3drepo — APPROVED** [review]

**Pull request merged**